packages/nimble-components/src/combobox/index.ts

```diff
@@ -32,32 +50,52 @@ declare global {
  * A nimble-styed HTML combobox
  */
 export class Combobox
-    extends FoundationCombobox
+    extends FormAssociatedCombobox
     implements DropdownPattern, ErrorPattern {
     @attr
     public appearance: DropdownAppearance = DropdownAppearance.underline;
 
-    /**
-     * The ref to the internal dropdown button element.
-     *
-     * @internal
-     */
-    @observable
-    public readonly dropdownButton?: ToggleButton;
-
     /**
      * A message explaining why the value is invalid.
-     *
-     * @public
-     * @remarks
-     * HTML Attribute: error-text
      */
     @attr({ attribute: 'error-text' })
     public errorText?: string;
 
     @attr({ attribute: 'error-visible', mode: 'boolean' })
     public errorVisible = false;
 
+    /**
+     * The autocomplete attribute.
+     */
+    @attr({ attribute: 'autocomplete', mode: 'fromView' })
+    public autocomplete: ComboboxAutocomplete | undefined;
+
+    /**
+     * The placement for the listbox when the combobox is open.
+     */
+    @attr({ attribute: 'position' })
+    public positionAttribute?: SelectPosition;
+
+    /**
+     * The open attribute.
+     */
+    @attr({ attribute: 'open', mode: 'boolean' })
+    public open = false;
+
+    /**
+     * Sets the placeholder value of the element, generally used to provide a hint to the user.
+     */
+    @attr
+    public placeholder = '';
```

Comment:
Can we avoid setting the placeholder to empty string? All `@att` marked properties will end up being reflected to the attribute which isn't what we want here. i.e. we end up with `<nimble-combobox placeholder="">`.

This is a [change in behavior from fast](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/combobox/combobox.ts#L199)

```suggestion
    public placeholder?: string;
```